src/pages/unseoJang/Main/Feed.js

```diff
@@ -0,0 +1,273 @@
+import React, { Component } from 'react';
+
+import Comment from './Comment';
+
+class Feed extends Component {
+  constructor(props) {
+    super(props);
+    //state 생성
+    this.state = {
+      commentValue: '',
+      commentList: [],
+      value: '', //this.state를 다시 빈 분자열로
+    };
+  }
+
+  //state 상태 핸들러
+  handleCommentInput = e => {
+    this.setState({
+      commentValue: e.target.value,
+      value: e.target.value,
+    });
+  };
+  //버튼 클릭 이벤트와 함수 생성
+  addComment = e => {
+    const { commentList } = this.state; //구조분해할당
+    this.setState({
+      commentList: commentList.concat({ comment: this.state.commentValue }),
+      commentValue: '',
+      value: '',
+    });
```

Comment:
setState를 기존의 state를 이용해서 새로운 값을 만들어서 변화시킬때 this.state로 가져오셔도 되지만! setState의 인자를 함수형태로 넣으면 prevState를 인자로 넘겨줍니다. 그렇기에 this.state에 의존하지 않고 바로 이전 state를 가져와서 다음 state를 만드는데 활용할 수 있습니다!
```suggestion
this.setState((prevState) => {
  return {
    commentList: prevState.commentList.concat({ comment: prevState.commentValue }),
    commentValue: '',
    value: '',
  };
});
```